Rules/rules.css

```diff
@@ -0,0 +1,12 @@
+.rule-book{
+    color: var(--card-text-color);
+    margin: 2rem auto;
+    width: 40rem;
```

Comment:
suggestion - You can use percentages for responsiveness

Reply:
I will make it responsive using media queries. This is for submissions only.